include/engine/datafacade/contiguous_internalmem_datafacade.hpp

```diff
@@ -890,6 +910,40 @@ class ContiguousInternalMemoryDataFacadeBase : public BaseDataFacade
     {
         return edge_based_node_data.IsSegregated(id);
     }
+
+    std::vector<extractor::ManeuverOverride>
+    GetOverridesThatStartAt(const NodeID edge_based_node_id) const override final
+    {
+        std::vector<extractor::ManeuverOverride> results;
+
+        bool found = false;
+        // Copy all the overrides into the results array.
+        // m_maneuver_overrides is sorted by node_sequence.front(),
+        // so we can stop searching after we get to a node > edge_based_node_id
+        for (const auto & override : m_maneuver_overrides)
```

Comment:
Good catch - yeah, it would be faster, `std::equal_range` has sub-linear performance if the thing being searched is random-access (as opposed to just forward iterable).

The way we're comparing a member variable (`override.start_node`) doesn't fit with the `std::equal_range` pattern though, I'll see if I can re-jigger it to work.